packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -77,81 +79,90 @@ const EDiscovery = WebexPlugin.extend({
-   * @param {int} size - Number of records to retrieve
+   * @param {Object} page - object with offset and size parameters indicating the start position from which to retrieve
+   *                        content and the number of activities to retrieve
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity<Array<ReportRecord>>>} Http Response containing a list of report records
    */
-  getReports({offset, size} = {}) {
+  getReports(page, timeoutMs = 10000) {
     return this.request({
       method: 'GET',
       service: 'ediscovery',
       resource: 'reports',
-      qs: {offset, size}
+      qs: {offset: page.offset, size: page.size},
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Deletes a specified report
    * @param {UUID} reportId - Id of the report being requested for deletion
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity>} HttpResponse indicating if delete was successful
    */
-  deleteReport(reportId) {
+  deleteReport(reportId, timeoutMs = 10000) {
     if (!reportId) {
       throw Error('Undefined parameter');
     }
 
     return this.request({
       method: 'DELETE',
       service: 'ediscovery',
-      resource: `reports/${reportId}`
+      resource: `reports/${reportId}`,
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Restarts a completed or cancelled report so that it begins again from scratch
    * @param {UUID} reportId - Id of the report being requested
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity<ReportRecord>>} Http response containing the report record
    */
-  restartReport(reportId) {
+  restartReport(reportId, timeoutMs = 10000) {
     if (!reportId) {
       throw Error('Undefined parameter');
     }
 
     return this.request({
       method: 'PUT',
       service: 'ediscovery',
-      resource: `reports/${reportId}`
+      resource: `reports/${reportId}`,
+      timeout: timeoutMs
     });
   },
 
   @waitForValue('@')
   /**
    * Retrieves content associated with a report
    * @param {UUID} reportId - Id of the report which contains the content
-   * @param {int} offset - Start position from which to retrieve content
-   * @param {int} size - Number of activities to retrieve
+   * @param {Object} page - object with offset and size parameters indicating the start position from which to retrieve
+   *                        content and the number of activities to retrieve
+   * @param {int} timeoutMs - optional connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity<[Activity]>>} Http response containing the activities
    */
-  getContent(reportId, {offset, size} = {}) {
+  getContent(reportId, page, timeoutMs = 10000) {
```

Comment:
I change the object literal to be a named parameter as i saw a test failure, but looking over now i see i accidentally removed the default value for this object `= {}`, so i'm going to fix this and see if it resolves the test failure also

Reply:
yes, switching back to `{offset, size} = {}` works fine, will go with that